library/ix-dev/community/adguard-home/upgrade_strategy

```diff
@@ -6,14 +6,14 @@ import sys
 from catalog_update.upgrade_strategy import semantic_versioning
 
 
-RE_STABLE_VERSION = re.compile(r'v[0-9]+\.[0-9]+\.[0-9]+')
+RE_STABLE_VERSION = re.compile(r'v\d+\.\d+\.\d+')
```

Comment:
```suggestion
RE_STABLE_VERSION = re.compile(r'v[0-9]+\.[0-9]+\.[0-9]+')
```

Reply:
Could you explain why `[0-9]` should be used instead of `\d`? And also the above `if not version` line, why `if version is None` shouldn't be used?

Reply:
Mostly for consistency with the other apps.
And also I don't know if the specific regex language has support for regex class
I'll let @Qubad786 answer this (and the `if not version` vs `if version is None`) as python is he's area of expertise!

Reply:
@xxyzz it is fine to use `\d` instead as it's the same thing.
About `version is None` - if you are explicitly looking out for `None` then it is indeed better to check for that explicitly instead of a boolean check i.e `not version`